src/hotspot/share/gc/shared/referenceProcessorPhaseTimes.hpp

```diff
@@ -80,7 +80,7 @@ class ReferenceProcessorPhaseTimes : public CHeapObj<mtGC> {
   ReferenceProcessorPhaseTimes(GCTimer* gc_timer, uint max_gc_threads);
   ~ReferenceProcessorPhaseTimes();
 
-  WorkerDataArray<double>* phase2_worker_time_sec() const { return _phase2_worker_time_sec; }
+  WorkerDataArray<double>* SoftWeakFinalRefsPhase_worker_time_sec() const { return _SoftWeakFinalRefsPhase_worker_time_sec; }
```

Comment:
This is a weird function name.  I understand why, but I'm not convinced.  We have process_soft_weak_final_refs and the like, for example.

Reply:
How about `soft_weak_final_refs_phase_worker_time_sec()`?

Reply:
Yes, that's normal style.  It's quite a mouthful, but it's not like it gets written very often.  With that change, I think this PR looks good.

Reply:
Agree with that.